### DISCUSSION TITLE
OPG

### DISCUSSION THREAD

**Discussion by vectorkovacspeter** [nav-gov-hu/eVAT#6] [Kérdések és válaszok - Gépi interfész] [answered]
Tisztelt illetékesek,

mikorra várható a dokumentációban is emlegetett OPG elérhetőségége (2021.Q2 -re ígérték, majd 2021.Q4 -ben jelezék, hogy teljesen megváltozik, mert KOBAK helyett Online számla rendszerből dolgozik majd)? 

Mert a hivatkozott https://github.com/nav-gov-hu/Online-Cash-Register-Logfile repóban ~3 éve nem történt semmi és nagyon nem lenne előnyös, ha egyszerre vezetnék be az eVAT -ot és az OPG API-t is, pláne, ha ez eVAT épít is OPG-ből származó adatokra.

**Comment by Edi1978** [comment]
Nekünk pár napja azt írták, hogy célegyenesben van. Mi is nagyon várjuk!

**Comment by renced42** [comment, marked as answer]
Kedves @vectorkovacspeter 
Frissítettük az https://github.com/nav-gov-hu/Online-Cash-Register-Logfile  repot és a mai napon megjelenik az ezzel a témával kapcsolatos közlemény is a NAV honlapján.
